Homework/src/by/tms/lesson12/additionalTasks/task_1/Task_1.java

```diff
@@ -11,19 +11,19 @@
     public static void main(String[] args) throws IOException {
         BufferedReader reader = null;
         try {
-            reader = new BufferedReader(new FileReader("/Users/trifonovanv/Desktop/TeachMeSkills/" +
-                    "TMS_Homework/Homework/src/by/tms/Lesson_12/additionalTasks/task_1/input.txt"));
+            reader = new BufferedReader(new FileReader("src/by/tms/lesson12/additionalTasks/task_1/input.txt"));
         } catch (FileNotFoundException e) {
             System.err.println("There is no such file!");
+            reader.close();
```

Comment:
reader still will not be closed in case of no exception appears. And even appears it could be null and you'll get another exception (NullPointerException)